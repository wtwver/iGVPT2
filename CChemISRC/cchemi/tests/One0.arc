

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020

           Empirical Formula: H2 O  =     3 atoms

 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36282 EV
          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          DIPOLE                  =          1.73858 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.020 SECONDS


          FINAL GEOMETRY OBTAINED
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870100 +1  -0.44376600 +1
  H     0.00000000 +1  -0.75754000 +1  -0.46039600 +1
  O     0.00000000 +1   0.00472800 +1   0.10883100 +1
 
