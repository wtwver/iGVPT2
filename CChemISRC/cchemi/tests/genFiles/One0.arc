

                     SUMMARY OF PM3 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 15.146L)
                                                       Sat Dec 12 23:24:34 2015
                                                       No. of days left = 165

           Empirical Formula: H2 O  =     3 atoms

PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43282 KCAL/MOL =    -223.56293 KJ/MOL
          TOTAL ENERGY            =       -324.90736 EV
          ELECTRONIC ENERGY       =       -474.28723 EV
          CORE-CORE REPULSION     =        149.37987 EV
          GRADIENT NORM           =          0.14144
          DIPOLE                  =          1.73859 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316494 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.059
          MOLECULAR WEIGHT        =         18.015
          COSMO AREA              =         42.47 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.19 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     1.53592
            H     2    O     1     0.56096
            O     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.010 SECONDS


          FINAL GEOMETRY OBTAINED
PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  O    -0.00001000 +1  -0.38096200 +1   0.00000000 +1
  H     0.76796600 +1   0.17999600 +1   0.00000000 +1
  H    -0.76792600 +1   0.17996400 +1  -0.01000000 +1
 
