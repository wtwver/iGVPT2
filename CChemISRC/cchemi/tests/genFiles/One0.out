 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 165    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM3 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Sat Dec 12 23:24:34 2015  *
 *  PM3        - The PM3 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GRADIENTS- ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O         -0.00001000  *  -0.38096200  *   0.00000000  *
     2       H          0.76796600  *   0.17999600  *   0.00000000  *
     3       H         -0.76792600  *   0.17996400  *  -0.01000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O       -0.0000   -0.3810    0.0000
     2         H        0.7680    0.1800    0.0000
     3         H       -0.7679    0.1800   -0.0100

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads = 16                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4


 -------------------------------------------------------------------------------
PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM3 CALCULATION
                                                       MOPAC2012 (Version: 15.146L)
                                                       Sat Dec 12 23:24:34 2015
                                                       No. of days left = 165




          FINAL HEAT OF FORMATION =        -53.43282 KCAL/MOL =    -223.56293 KJ/MOL


          TOTAL ENERGY            =       -324.90736 EV
          ELECTRONIC ENERGY       =       -474.28723 EV  POINT GROUP:     C2v 
          CORE-CORE REPULSION     =        149.37987 EV
          COSMO AREA              =         42.47 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.19 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.14144
          IONIZATION POTENTIAL    =         12.316494 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.059
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.015

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     1.53592
            H     2    O     1     0.56096
            O     1    H     2     0.00000


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.009 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE       GRADIENT
      1          1  O    CARTESIAN X    -0.000010    -0.013528  KCAL/ANGSTROM
      2          1  O    CARTESIAN Y    -0.380962    -0.063529  KCAL/ANGSTROM
      3          1  O    CARTESIAN Z     0.000000     0.000563  KCAL/ANGSTROM
      4          2  H    CARTESIAN X     0.767966     0.086386  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y     0.179996     0.009443  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z     0.000000     0.000464  KCAL/ANGSTROM
      7          3  H    CARTESIAN X    -0.767926    -0.072858  KCAL/ANGSTROM
      8          3  H    CARTESIAN Y     0.179964     0.054086  KCAL/ANGSTROM
      9          3  H    CARTESIAN Z    -0.010000    -0.001027  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O         -0.00001000  *  -0.38096200  *   0.00000000  *
     2       H          0.76796600  *   0.17999600  *   0.00000000  *
     3       H         -0.76792600  *   0.17996400  *  -0.01000000  *

                             CARTESIAN COORDINATES

   1    O       -0.000010000    -0.380962000     0.000000000
   2    H        0.767966000     0.179996000     0.000000000
   3    H       -0.767926000     0.179964000    -0.010000000


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -36.82439 -17.58089 -14.52270 -12.31649   4.05945   5.33163


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          O          -0.358559        6.3586     1.76337     4.59519
    2          H           0.179279        0.8207     0.82072
    3          H           0.179280        0.8207     0.82072
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.966    -0.009     0.966
 HYBRID         0.000     0.772    -0.007     0.772
 SUM            0.000     1.739    -0.015     1.739


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  O   1.76337   1.17967   1.41560   1.99992
    2  H   0.82072
    3  H   0.82072

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
